release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -514,6 +514,12 @@ Otherwise, alert receivers that use the files will fail to deliver notifications
 [id="ocp-4-12-scalability-and-performance"]
 === Scalability and performance
 
+==== Mount namespace encapsulation (Technology Preview)
+
+Encapsulation is the process of moving all Kubernetes-specific mount points to an alternative namespace to reduce the visibility and performance impact of a large number of mount points in the default namespace. Previously, mount namespace encapsulation has been deployed transparently in OpenShift Container Platform specifically for Distributed Units (DUs) installed using the ZTP process. In {product-title} {product-version}, this proven functionality is being rolled out to all customers as a configurable option.
+
+A standard host operating system uses systemd to constantly scan all mount namespaces: both the standard Linux mounts and the numerous mounts that Kubernetes uses to operate. The current implementation of Kubelet and CRI-O both use the top-level namespace for all container and Kubelet mountpoints. Encapsulating these container-specific mountpoints in a private namespace reduces systemd overhead and enhances CPU performance. Encapsulation can also improve security, by storing Kubernetes-specific mount points in a location safe from inspection by unprivileged users.
```

Comment:
I think an xref to the relevant docs would mean this paragraph isn't necessary.

Reply:
I take the opposite view. 
Para 1 announces, para 2 describes. I'd rather do the describing here rather than relying on readers following a link.
Jim to decide.